raiden/transfer/state.py

```diff
@@ -456,11 +459,12 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'address': to_checksum_address(self.address),
             'token_address': to_checksum_address(self.token_address),
             'network_graph': self.network_graph,
-            'partneraddresses_to_channels': map_dict(
+            'channelidentifiers_to_channels': map_dict(
                 to_checksum_address,
```

Comment:
the channel identifier is an int, so you can remove the whole `map_dict(...)` here.

Reply:
please, check it again